simpeg/directives/_regularization.py

```diff
@@ -403,17 +403,22 @@ def stopping_criteria(self):
                 )
             return True
 
-        # Check if the function has changed enough
-        f_change = np.abs(self.metrics.f_old - phim_new) / (self.metrics.f_old + 1e-12)
+        # Get how much the regularization has changed from previous value
+        f_change = (
+            np.abs(self.metrics.f_old - phim_new) / self.metrics.f_old
+            if self.metrics.f_old != 0.0
+            else np.inf
+        )
+
+        # Get target chi factor minus current chi factor
+        chi_factor_diff = 1.0 - self.invProb.phi_d / self.misfit_from_chi_factor(
+            self.chifact_target
+        )
 
         if (
             f_change < self.f_min_change
             and self.metrics.irls_iteration_count > 1
-            and np.abs(
-                1.0
-                - self.invProb.phi_d / self.misfit_from_chi_factor(self.chifact_target)
-            )
-            < self.misfit_tolerance
+            and (0 <= chi_factor_diff < self.misfit_tolerance)
```

Comment:
The absolute value was intended to allow to be "within range" of the target. That is, it's ok if the misfit is slightly higher or lower than the target. 
Forcing it to be strictly smaller is a bit more restrictive.

Reply:
Right, not using the absolute value would make the stopping criteria more restrictive. The reason of the change is that it sounds unintuitive that it's possible to get a model that generates a chifact greater than the target.

Talking with @lheagy, one option would be to offer a flag to allow users to make the stopping criteria less restrictive by using the abs, but not using the abs by default. What do you think?

Reply:
I have no objection. The only risk I can foresee is an IRLS that runs longer, but ending below the target makes sense.
We can make the change, and revert the default later if we see instability.